ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -153,6 +153,10 @@ class CellEditorOverlay extends Component {
       ..._.mapValues(queryModifiers, qm => this.queryStateReducer(qm)),
     }
 
+    const querySavable = query =>
```

Comment:
Definitely.  Anything that returns a bool in this app has the 'is' prefix.  Will change function to reflect